package.json

```diff
@@ -10,5 +10,9 @@
   "devDependencies": {
     "@playwright/test": "^1.40.1",
     "@types/node": "^20.10.4"
+  },
+  "dependencies": {
+    "dotenv": "^16.3.1",
+    "save-dev": "^0.0.1-security"
```

Comment:
not sure if we need this dependency here. 
Also try to save all dependencies into devDependencies